data/json/items/armor/helmets.json

```diff
@@ -163,9 +165,10 @@
     "symbol": "[",
     "looks_like": "hat_hard",
     "color": "blue",
-    "covers": [ "HEAD" ],
-    "coverage": 95,
-    "encumbrance": 20,
+    "armor_portion_data": [
+      { "covers": [ "head" ], "coverage": 95, "encumbrance": 20 },
+      { "covers": [ "mouth", "eyes" ], "coverage": 50, "encumbrance": 0 }
```

Comment:
50 coverage seems a little high, especially for the mouth.

Reply:
I was thinking of a baseball helmet with the mesh on the front, similar to football helmet. If its not this style (the pic you linked isnt) then I will simply remove all the eye and mouth coverage. What do you think?

Reply:
I think the style without the mesh is more common, but have no strong opinion.